api/HTMLContentElement.json

```diff
@@ -5,13 +5,16 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLContentElement",
         "support": {
           "chrome": {
-            "version_added": "35"
+            "version_added": "35",
+            "version_removed": "89"
           },
           "chrome_android": {
-            "version_added": "37"
+            "version_added": "37",
+            "version_removed": "89"
           },
           "edge": {
-            "version_added": "≤79"
```

Comment:
Changing ≤79 to 79 was intentional, suggested by mdn-bcd-collector which means the interfaces weren't in Edge 18.